homeassistant/components/flux/switch.py

```diff
@@ -193,12 +226,49 @@ def __init__(
         mode,
         interval,
         transition,
-    ):
+    )
+
+    async_add_entities([flux])
+
+    platform = entity_platform.async_get_current_platform()
+
+    async def async_update() -> None:
+        """Update lights."""
+        await flux.async_flux_update()
+
+    platform.async_register_entity_service(
+        "flux_update",
+        {},
+        async_update,
+    )
```

Comment:
Can be removed, when migrated to config entries.

Reply:
@jpbede is this comment still relevant? If so can you explain why this can be removed?

Reply:
Ok, nvm, this should be kept. Thought it would be a reload service, to update the config :)

Reply:
As far as I can see all comments are resolved now. Can you take a second look at the code?